mod/aalborg_theme/views/default/elements/forms.css.php

```diff
@@ -250,6 +250,16 @@
 	background-color: #EEE;
 	display: block;
 }
+.ui-helper-hidden-accessible {
+	border: 0;
+	clip: rect(0 0 0 0);
+	height: 1px;
+	margin: -1px;
+	overflow: hidden;
+	padding: 0;
+	position: absolute;
+	width: 1px;
```

Comment:
I assume this been copied from somewhere? I'm wondering if all these are necessary.